MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -3806,11 +3800,11 @@ public Money getWeeklyMaintenanceCosts() {
     }
 
     public Money getOverheadExpenses() {
-        if(!campaignOptions.payForOverhead()) {
+        if (getCampaignOptions().payForOverhead()) {
```

Comment:
Same here, not really sure of the point of rearranging the if block.

Reply:
This is to prevent potential issues from the statement being false first and true second (I don't really care, but I've seen this result in bugs from lazy devs who didnt check for a false statement first more often than you'd expect)